app/src/main/java/com/example/cocktailmaster/utils/GlideHelper.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.cocktailmaster.utils
+
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.example.cocktailmaster.R
+
+fun ImageView.loadImage(url: String) {
+    Glide.with(context).load(url)
+        .error(R.drawable.image_error)
```

Comment:
Đổi tên file là ImageExtension hoặc GlideExtension
```suggestion
fun ImageView.loadImage(url: String) {
    Glide.with(context)
        .load(url)
        .error(R.drawable.image_error)
```